assets/css/app.css

```diff
@@ -168,34 +168,59 @@ body {
 
 .auction-title {
     margin: 0;
-    font-family: Roboto, sans-serif;
     font-weight: 600;
     height: 2.5rem;
 }
 
 .auction-description {
     font-style: italic;
-    font-family: Roboto, sans-serif;
     font-size: 1em;
     margin-bottom: 0.5rem;
     border-bottom: #BDBDBD solid 1px;
     padding-bottom: 0.5rem;
 }
 
+.auction-container li {
+    margin: 0;
+    font-weight: 400;
+    list-style-type: none;
+}
+
 .auction-detail {
     margin: 0;
-    font-family: Roboto, sans-serif;
     font-weight: 400;
     list-style-type: none;
 }
 
-@media only screen and (max-width: 60.0rem) {
+.show-auction {
+    overflow: -webkit-paged-x;
+}
+.show-auction.content-text {
+    font-size: 1.37em;
+}
+
+.centered-container {
+    margin: 1rem 2rem 1rem 2rem;
+    background-color: #FAFAFA;
+    padding: 1rem 1.5rem;
+    border-radius: 4px;
+    transition: all 0.5s cubic-bezier(0.25,0.8,0.25,1);
+    list-style-type: none;
+    font-size: 0.7em;
+}
 
+.countdown {
+    font-size: 0.8em;
+    padding-bottom: 2rem;
+    text-align: -webkit-center;
+}
+
+@media only screen and (max-width: 60.0rem) {
```

Comment:
usually for max width we use px because that can correspond with mobile/desktop easier.